bpf/tests/prog_test/prog_test.go

```diff
@@ -12,8 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// NB: leave this out from the CI for now.
-// +build noci
+// +build privileged_tests
```

Comment:
This is not enough to enable the test in CI. Changes are also required at https://github.com/cilium/cilium/blob/90c1acaa557edd101fd0e3bd409f9c643a88efda/Makefile#L34
I didn't implement those changes because the test then failed with some incorrect opcode error. Additional work will be required to enable this in CI.

Reply:
Tracked at https://github.com/cilium/cilium/issues/14907.